app/charts/map/map.tsx

```diff
@@ -255,12 +237,40 @@ export const MapComponent = () => {
           <GeoJsonLayer
             id="areas"
             data={features.areaLayer}
-            pickable={false}
+            pickable={true}
             stroked={true}
             filled={true}
             extruded={false}
             lineWidthMinPixels={0.5}
             lineWidthMaxPixels={1}
+            onHover={({
+              x,
+              y,
+              object,
+            }: {
+              x: number;
+              y: number;
+              object: GeoShapeFeature;
+            }) => {
+              if (object) {
+                dispatch({
+                  type: "INTERACTION_UPDATE",
+                  value: {
+                    interaction: {
+                      visible: true,
+                      mouse: { x, y },
+                      d: data.find(
+                        (d) => getLabel(d) === object.properties.label
+                      ),
```

Comment:
Mmh, again an index seems like it could be good, I am worried for performance here since we need to find the object in a list with each movement of the mouse. Since we have the object being hovered  in the callback, couldn't we "save" the datum inside this object when rendering so that we can just do `interaction: { object.d }` ?

Reply:
Yes, this kind of pattern will be changed, I marked some of it with FIXME label 😄 Probably it makes much more sense to save the data inside the object to avoid doing a search.